tokio/tests/io_fill_buf.rs

```diff
@@ -1,5 +1,5 @@
 #![warn(rust_2018_idioms)]
-#![cfg(feature = "full")]
+#![cfg(all(feature = "full", not(target_os = "wasi")))]
 
 use tempfile::NamedTempFile;
```

Comment:
It's unfortunate that this test is written to use a file. It means that `fill_buf` is not tested on WASM.

You don't have to fix this in your PR.